conf/conf.yaml

```diff
@@ -27,4 +27,4 @@ discovery:                       # service discovery center
       - "127.0.0.1:2181"
     prefix: /zookeeper
     weight: 100
-    timeout: 60
+    timeout: 10
```

Comment:
Why reduce the timeout here?

Reply:
Let's add time unit in the comment.

Reply:
comment added, abuot timeout, I think 60 seconds was too long, so shortened it.